### ISSUE TITLE
"Notifications not supported" while connecting to remote geth node via rpc

### ISSUE THREAD

**Issue by rrenim** [ethereum/mist#4024]
- [v] I've asked for help in the [Mist Gitter](http://gitter.im/ethereum/mist) before filing this issue.

```
Version: `0.11.1` (Ethereum Wallet)
OS & Version: windows 10
Node version: `geth 1.8.12`
Number of blocks synchronized: 0
```

Starting Ethereum Wallet with remote Geth node (fully synced) gives an error:
```
[ERROR] ipcProviderBackend - [39mSend request failed { code: -32000, message: 'notifications not supported' }
```
GUI appears, but its state isn't refreshed after start.
Ethereum Wallet start script:
```
Ethereum Wallet.exe  --rpc http://xxx.xxx.xxx.xxx:port
```
geth start script (on remote Ubuntu host):
```
/usr/bin/geth --fast --cache=2048 --rpc --rpcport [port] --rpcapi "admin,debug,miner,shh,txpool,personal,eth,net,web3" --rpcaddr 0.0.0.0  
```
Ethereum Wallet 0.10.0 with the same node doesn't give this error.

p.s. Connection to node is established using vpn tunnel.

**Comment by ryanio** [comment]
Hey .Connection over HTTP is deprecated due to the need for Pub-Sub subscriptions, so your options are either WebSockets (`--ws`) or IPC (default).

**Comment by rrenim** [comment]
Thank's for the reply, Ryano :)
I also read, that http becomes deprecated. But I still don't know - how to instantiate websocket-connection from Ethereum Wallet. "ws://" prefix in address and "--ws" key don't work.
It would be nice to get sample of launching EthWallet with remote geth-node using websockets.

**Comment by qbasso** [comment]
I'm experiencing exact same isue, using "--ws", "--wsaddr", "--wsport" with `geth` command and supplying "ws://" prefixed url to Mist does not work.

**Comment by wby3902433** [comment]
I'm experiencing exact same issue

**Comment by natemiller1** [comment]
Went through the same thing, got it to work: Make sure port 8546 is open (8545 still opens by default for http/rpc), don't connect to ws:// still use http://

**Comment by rrenim** [comment]
Unfortunately natemiller1's trick doesn't work for me :( Mist launches a new local geth node instead of connecting to my remote one.

**Comment by JonasPf** [comment]
I'm having the same problem. Tried different parameters but it didn't work.

@ryanio I couldn't find the "--ws" parameter in the code (https://github.com/ethereum/mist/blob/develop/modules/settings.js#L337) and it doesn't show it to me when I call "mist --help". Does it exist?

According to this code (https://github.com/ethereum/mist/blob/develop/modules/settings.js#L131) websockets are not supported. Is that correct?

**Comment by kbs1** [comment]
Yeah, websocket connections are still not supported, with `--rpc "ws://localhost:8546"` I get 
```
[2019-02-24T02:39:56.338] [WARN] Settings - Websockets are not yet supported by Mist, using default IPC connection
```
But with IPC connection it still tries to use websockets for number of blocks updating etc. This results in 
```
RangeError: Invalid WebSocket frame: invalid status code 1006
``` (see #4176)

So all in all, it doesn't work :(

**Comment by stvenyin** [comment]
xx

**Comment by sssubik** [comment]
Hey, I tried using mist in remote server with UI open in the local computer.
I get this:
`EthereumNodeRemote - Remote websocket connection not open, attempting to reconnect and retry eth_subscribe...`
How do I connect to remote IPC or Websocket? Is the issue over?
